packages/dapp/src/components/listing/ListingRedux.tsx

```diff
@@ -221,6 +241,38 @@ class ListingPageComponent extends React.Component<
   private showCharterTab = (): void => {
     this.onTabChange(0);
   };
+
+  private closeStoryBoost = () => {
+    let urlBase = this.props.location.pathname;
+    urlBase = urlBase.substring(0, urlBase.indexOf("/"));
+    this.props.history.push({
+      pathname: urlBase + "/listing/" + this.props.listingId + "/storyfeed",
+    });
+  };
+
+  private openStoryDetails = (postId: string) => {
+    let urlBase = this.props.location.pathname;
+    urlBase = urlBase.substring(0, urlBase.indexOf("/"));
+    this.props.history.push({
+      pathname: urlBase + "/listing/" + this.props.listingId + "/storyfeed/" + postId,
+    });
+  };
+
+  private openPayments = (postId: string) => {
+    let urlBase = this.props.location.pathname;
+    urlBase = urlBase.substring(0, urlBase.indexOf("/"));
+    this.props.history.push({
+      pathname: urlBase + "/listing/" + this.props.listingId + "/storyfeed/" + postId + "/payment",
+    });
+  };
+
+  private openStoryNewsroomDetails = (postId: string) => {
+    let urlBase = this.props.location.pathname;
+    urlBase = urlBase.substring(0, urlBase.indexOf("/"));
+    this.props.history.push({
+      pathname: urlBase + "/listing/" + this.props.listingId + "/storyfeed/" + postId + "/newsroom",
+    });
+  };
```

Comment:
Same here - If the newsroom modal is blocked from rendering on listing page you don't need this.